src/services/jsonforms.ts

```diff
@@ -146,10 +146,9 @@ const generateUISchema = (
     }
 
     const types = deriveTypes(jsonSchema);
-    if (types.length === 0) {
-        // TODO - this was returning null initially but the compiler complained so I put this in here to make it be nice.
-        return { type: 'Type Missing on UI Schema' };
-    }
+    // if (types.length === 0) {
+    //   return null;
+    // }
```

Comment:
Instead of changing code that is being brought in from 3rd parties if we don't need to worry about something (like getting sent crappy schemas) I think commenting stuff out is best. Makes it a bit easier to compare to the original when the time comes.